src-cpp/rdkafkacpp.h

```diff
@@ -2649,6 +2634,22 @@ class RD_EXPORT KafkaConsumer : public virtual Handle {
   virtual ConsumerGroupMetadata *groupMetadata () = 0;
 
 
+  /** @brief Check whether the consumer considers the current assignment to
+   *         have been lost involuntarily. This method is only applicable for
+   *         use with a subscribing consumer. Assignments are revoked
+   *         immediately when determined to have been lost, so this method is
+   *         only useful within a rebalance callback. Partitions that have
+   *         been lost may already be owned by other members in the group and
+   *         therefore commiting offsets, for example, may fail.
+   *
+   * @remark Calling rd_kafka_assign(), rd_kafka_incremental_assign() or
```

Comment:
This should reference the C++ API names (and make sure to run `make docs` to verify that Doxygen is able to pick them up correctly (it will warn otherwise I think)).